applications/nrf_desktop/src/modules/Kconfig.dvfs_state_template

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig DESKTOP_DVFS_STATE_$(dvfs_state)_ENABLE
+	bool "Handle DESKTOP_DVFS_STATE_$(dvfs_state)"
+	default y
+	depends on $(dvfs_dependencies)
+
+if DESKTOP_DVFS_STATE_$(dvfs_state)_ENABLE
+
+if $(dvfs_timeout) != 0
+
+config DESKTOP_DVFS_STATE_$(dvfs_state)_TIMEOUT_MS
+	int "timeout"
+	default $(dvfs_timeout)
+	range 0 65535
+	help
+	  Timeout in milliseconds specifying time after which DVFS request related to
+	  DESKTOP_DVFS_STATE_$(dvfs_state) times out.
+	  A request is automatically cleared on timeout.
+
+endif # $(dvfs_state)_TIMEOUT_MS != 0
+
+if $(dvfs_timeout) = 0
+
+config DESKTOP_DVFS_STATE_$(dvfs_state)_TIMEOUT_MS
+	int
+	default 0
+	range 0 0
+	help
+	  Timeout in milliseconds specifying time after which DVFS request related to
+	  DESKTOP_DVFS_STATE_$(dvfs_state) times out.
+	  A request is automatically cleared on timeout.
+
+endif # $(dvfs_state)_TIMEOUT_MS = 0
+
+choice "DESKTOP_DVFS_STATE_$(dvfs_state)_ACTIVE_FREQ"
```

Comment:
```suggestion
choice DESKTOP_DVFS_STATE_$(dvfs_state)_ACTIVE_FREQ
```

Reply:
that is very strange, see other occurrences of choice 

https://github.com/nrfconnect/sdk-nrf/blob/main/subsys/bluetooth/services/fast_pair/use_case/Kconfig#L7

could you investigate why this is the case?

Reply:
I didn't dig into it just saw that in other places where we use templates and choice we have such notation: https://github.com/nrfconnect/sdk-zephyr/blob/main/subsys/logging/Kconfig.template.log_config#L3

Reply:
I see, I guess it is necessary addition in case of a Kconfig template.